pkg/convert/testdata/programs/sandbox_multiple_registry_modules/pcl/main.pp

```diff
@@ -0,0 +1,30 @@
+package "subnets" {
+  baseProviderName    = "terraform-module"
+  baseProviderVersion = "0.1.3"
+  parameterization {
+    name    = "subnets"
+    version = "1.0.0"
+    value   = "eyJtb2R1bGUiOiJyZWdpc3RyeS50ZXJyYWZvcm0uaW8vaGFzaGljb3JwL3N1Ym5ldHMvY2lkciIsInBhY2thZ2VOYW1lIjoic3VibmV0cyIsInZlcnNpb24iOiIxLjAuMCJ9"
```

Comment:
I'm still miffed this is unreadable. Does PCL have functions for base64-encoding? So we could splice a call to base64-encode a readable structure?

Reply:
>  so the closest way to represent that in text is base64-encoded string.

The extend to which we need to care I think is just the extent to which this is visible. Possibly PCL is not visible to the casual user converting to TypeScript so it is just an implementation detail then we do not care (just could be easier to debug if it was readable). I guess in the generated code the parameter value is not written anywhere yet right, at least at the moment? It is written/embedded into the generated SDK. There was some talk about writing it into Pulumi.yaml or similar, which would be another point observable by the user. It is those locations where users would "see" it that we could consider making more legible.

I guess we should punt on it for now unless it's really trivial here.

Reply:
>I guess in the generated code the parameter value is not written anywhere yet right, at least at the moment? It is written/embedded into the generated SDK.

That's right, it is only embedded in the generated parameterized SDK